test/test2.zoauthServer.js

```diff
@@ -7,19 +7,24 @@
 import { assert, expect } from "chai";
 import { setupLogger } from "zoapp-core";
 import zoauthServer from "../src/zoauthServer";
+import testConfig from "./config";
 
 setupLogger("test");
 
 const mysqlConfig = {
   database: {
-    datatype: "mysql",
-    host: "localhost",
-    name: "auth_test",
-    user: "root",
+    datatype: testConfig().database.datatype,
+    host: testConfig().database.host,
+    name: testConfig().database.name,
+    user: testConfig().database.user,
```

Comment:
do not use a function here. also, if you have the same config object in the other file, why are you creating a new config object here? can't you just use the same?

Reply:
You're true, i change that